src/map/modifier.h

```diff
@@ -601,6 +601,7 @@ enum class Mod
     EXP_BONUS         = 382,  //
     ROLL_RANGE        = 528,  // Additional range for COR roll abilities.
     JOB_BONUS_CHANCE  = 542,  // Chance to apply job bonus to COR roll without having the job in the party.
+    RANDOM_DEAL_BONUS = 573,  // % chance to reset 2 abilities
```

Comment:
We have a problem

Down in the file
`// 570 through 825 used by WS DMG mods these are not spares.`

Worst part is we have some of those IDs used when we shouldnt

Reply:
Oops, I'll find a new id

Reply:
Theres a list of spare Ids down bellow

Reply:
Updated, nice catch